sbysrc/sby.py

```diff
@@ -115,14 +134,18 @@ def __call__(self, parser, namespace, values, option_string=None):
         print("ERROR: Sby file does not have .sby file extension.", file=sys.stderr)
         sys.exit(1)
 
+elif initconfig:
+    print("default sby config written to default.sby", file=sys.stderr)
+    init_config("default.sby")
```

Comment:
Instead of `default.sby` (and `default.v`) use the command line arg for sby file (and same name with `.sv` instead of `.sby` for the verilog file).